nixos/modules/services/hardware/pcscd.nix

```diff
@@ -24,7 +24,6 @@ in
 
     plugins = mkOption {
       type = types.listOf types.package;
-      default = [ pkgs.ccid ];
       defaultText = literalExpression "[ pkgs.ccid ]";
```

Comment:
You probably want to remove this I'd say because it's not technically a default per se, is it?

Reply:
I guess that's fair but I don't see this pattern used :p cc @ncfavier who does a lot of work for nixos-search

Reply:
I think that's fine, in fact this pattern is why I made https://github.com/NixOS/nixpkgs/pull/230353. Not ideal obviously but I don't think we have a better way to indicate "the default value survives appending to the list with default override priority".